### DISCUSSION TITLE
Lunar Calendar Plasma Applet v0.2.0

### DISCUSSION THREAD

**Discussion by BLumia** [BLumia/pineapple-calendar#10] [General]
[Install Guide | 安装指南](https://github.com/BLumia/pineapple-calendar/wiki/%E5%AE%89%E8%A3%85%E6%8C%87%E5%8D%97)

This version is tested under Arch Linux and Ubuntu 22.04.

![image](https://user-images.githubusercontent.com/10095765/164233270-65d87635-a119-4ca6-8e71-919c3285aad5.png)

Notice: if you are using an older version of KDE Plasma desktop environment and v0.2.0 doesn't work, please consider checkout the v0.1.1 release at [here](https://github.com/BLumia/pineapple-calendar/releases/tag/0.1.1).

If you got any issues using the applet, please fire a bug or [create a discussion thread](https://github.com/BLumia/pineapple-calendar/discussions).

<hr /><em>This discussion was created from the release <a href='https://github.com/BLumia/pineapple-calendar/releases/tag/0.2.0'>Lunar Calendar Plasma Applet v0.2.0</a>.</em>

**Comment by mraandtux** [comment]
喜迎22.04，Deb打包版同时发布：[plasma-addon-pineapple-calendar_0.2.0-1.zip](https://github.com/BLumia/pineapple-calendar/files/8527024/plasma-addon-pineapple-calendar_0.2.0-1.zip)

**Reply by BLumia** [reply]
感谢更新，我也一并链接到了 [安装指南](https://github.com/BLumia/pineapple-calendar/wiki/%E5%AE%89%E8%A3%85%E6%8C%87%E5%8D%97) 中。
